internal/testing/testdata/testdata.go

```diff
@@ -420,200 +423,81 @@ var (
 	}
 
 	// CycloneDX Testdata
+	cdxTopLevelPack, _ = asmhelpers.PurlToPkg("pkg:oci/static@sha256:6ad5b696af3ca05a048bd29bf0f623040462638cb0b29c8d702cbb2805687388?repository_url=gcr.io/distroless/static&tag=nonroot")
```

Comment:
the top level package here has a digest, so we probably will need an artifact node as well with an `IsOccurrence` predicate to specify the relationship.

Reply:
I was thinking to address "occurrences" in the next iteration. The original implementation didn't have a support for them, so I thought to get "dependencies" right for starters.

Reply:
yea - that's fine. Would you be so kind to help open up an issue and link it as a TODO in the code? and should be LGTM to me.

Reply:
@lumjjb Done. Thanks